src/resolve/connectionResolver.ts

```diff
@@ -108,10 +109,32 @@ export class ConnectionResolver {
     if (query.type === defaultRegistry.types.standardvalueset.name && members.length === 0) {
       const standardValueSetPromises = standardValueSet.fullnames.map(async (standardValueSetFullName) => {
         try {
-          const standardValueSetRecord: StdValueSetRecord = await this.connection.singleRecordQuery(
-            `SELECT Id, MasterLabel, Metadata FROM StandardValueSet WHERE MasterLabel = '${standardValueSetFullName}'`,
-            { tooling: true }
-          );
+          // The 'singleRecordQuery' method was having connection errors, using `retry` resolves this
+          // Note that this type of connection retry logic may someday be added to jsforce v2
+          // Once that happens this logic could be reverted
+          const standardValueSetRecord: StdValueSetRecord = await retry(async () => {
```

Comment:
@iowillhoit why are there no limits being applied to the retry? The `retry` function defaults to retries and timeout of [INFINITELY](https://github.com/normartin/ts-retry-promise#configuration).

Reply:
@peternhale It looks like the `number | INFINITELY` in the docs are Types. The defaults are `10` retries and a `60s` for timeouts.